apps/mobile/src/packages/categories/libs/validation-schemas/validation-schemas.ts

```diff
@@ -0,0 +1 @@
+export { categoriesValidationSchema } from "shared";
```

Comment:
Be more specific with the name. Is it for getting categories, for saving, for updating?

Reply:
And one more important thing, if the form is fine, can I add the  `type Constructor` to `packages/shared` or use ` type ApiConstructor` as suggested by @katerynakostikova?